src/unstract/sdk/file_storage/env_helper.py

```
@@ -36,7 +36,7 @@ def get_storage(storage_type: StorageType, env_name: str) -> FileStorage:
             provider = FileStorageProvider(
                 file_storage_creds[CredentialKeyword.PROVIDER]
             )
-            credentials = file_storage_creds.get(CredentialKeyword.CREDENTIALS, "{}")
+            credentials = file_storage_creds.get(CredentialKeyword.CREDENTIALS, {})
```

@gaya3-zipstack It would be better to have it as string with an empty dict for fallback, else we might bump into error where it might expect a string, but we would have sent a dict.
Also, Is a default value needed here, or can we raise an exception if envs are not set? Sometimes, fallbacks are misleading and breaks when fs_instance invokes any method.

---

As discussed, having it as a string will break the type check of PermanentFileStorage and the like. So keeping it as dict.
At the moment, I do not see a need to pass credentials as empty for cases like local storage as it is not required. Hence not adding an exception for the same. However if "provider" is missing, we already are handling it as KeyError.